src/kbmod/result_list.py

```diff
@@ -920,11 +920,14 @@ def write_table(self, filename, overwrite=True):
             The name of the result file.
         overwrite : `bool`
             Overwrite the file if it already exists.
+        keep_all_stamps : `bool`
+            Keep individual stamps for each result and time step.
+            This is very expensive.
```

Comment:
Optional nit: since this is so expensive maybe add the obvious note to the docstring that it is False by default.